electrumsv/network.py

```diff
@@ -460,7 +460,9 @@ async def _negotiate_protocol(self):
         method = 'server.version'
         args = (PACKAGE_VERSION, [ version_string(PROTOCOL_MIN), version_string(PROTOCOL_MAX) ])
         try:
-            server_string, protocol_string = await self.send_request(method, args)
+            result = await self.send_request(method, args)
```

Comment:
Does this check all the possible cases? Or is it still possible for this user to poison the well with variations?

Reply:
i.e. get rid of the unsafe `None` check and just catch `TypeError` here.

Reply:
Done.